public/admin/index.html

```
@@ -44,30 +55,37 @@ <h1>local-hosted-wordle</h1>
-          <h2>Admin Console</h2>
-          <p class="note">
+          <h2 data-i18n="admin.title">Admin Console</h2>
+          <p class="note" data-i18n="admin.unlockHint">
             Enter your admin key to unlock provider tools. The key is session-scoped and never saved
             in browser storage.
           </p>
           <form id="unlockForm" class="create-form" autocomplete="off">
             <label>
-              Admin key
+              <span data-i18n="admin.unlock">Admin key</span>
               <input id="adminKeyInput" name="adminKey" type="password" autocomplete="off" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)